dev-packages/browser-integration-tests/suites/replay/customEvents/test.ts

```diff
@@ -47,7 +49,7 @@ sentryTest(
 
     expect(replayEvent0).toEqual(getExpectedReplayEvent({ segment_id: 0 }));
 
-    await page.locator('#img-button').click();
```

Comment:
Should we keep this?

Reply:
yes that click generates LCP, CLS and FID metrics and the next replay event

Reply:
I mean the `await`

Reply:
oh oops that should be kept, you're right